misc/newversion

```diff
@@ -43,16 +42,16 @@ umask 022
 
 OLD_EGGVERSION=$(grep AC_INIT configure.ac | sed -e 's/AC_INIT(\[Eggdrop\],\[//g' -e 's/\],\[bugs@eggheads.org\])//g')
 NEW_EGGVERSION=$(echo $OLD_EGGVERSION | cut -d. -f1-2).$(($(echo $OLD_EGGVERSION | cut -d. -f3) + 1))
-NEW_NUMVERSION=$(($(grep 'egg_numver =' src/main.c | sed -e 's/.*= //' -e 's/;$//') + 100))
+NEW_NUMVERSION=$(($(grep -E '^#define EGG_NUMVER.*' src/version.h |cut -d " " -f 3|sed -e 's/..$//' -e 's/$/00/') + 100))
```

Comment:
Why not use "sed -e 's/..$/00/'"? And I don't see any ERE's in that regex needing "grep -E"?